### PR TITLE
sql/ir: remove unused code

### PR DESCRIPTION
**Pull request by knz**

### CODE REVIEW

**Review by benesch — APPROVED** [review]

**Review by benesch — APPROVED** [review]

File: build/teamcity-testrace.sh [hunk 1]
```diff
@@ -24,7 +24,8 @@ else
 		echo "No changed packages; skipping race detector tests"
 		exit 0
 	fi
-	PKGSPEC=$(echo "${PKGSPEC}" | xargs -n1 dirname | sort | uniq | sed 's/^/.\//' | paste -sd " " -)
+	# Only keep the pathspecs for which there are go files remaining.
+	PKGSPEC=$(echo "${PKGSPEC}" | xargs -n1 dirname | sort | uniq | sed 's/^/.\//' | { while read path; do if ls "$path"/*.go >/dev/null 2>&1; then echo "$path"; fi; done; } | paste -sd " " -)
```
Comment (jordanlewis) [inline build/teamcity-testrace.sh]: does this work even when directories are fully removed? @danhhz ran into that recently

Comment (knz) [reply build/teamcity-testrace.sh]: Yes then the `ls` will fail and the thing is not listed.

Comment (knz) [reply build/teamcity-testrace.sh]: This PR is an example of it.

**Review by jordanlewis — APPROVED** [review]
Rest in peace `irgen` 🖖

**Comment by knz** [comment]
It's still very much alive and kicking on my computer. This kind of code is bound to pop up again sooner or later.

**Pull request merged**